### PR TITLE
fix(pingcap/ng-monitoring): fix rule file paths

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: packages/packages.yaml.tmpl [hunk 1]
```diff
@@ -261,22 +261,27 @@ components:
                 src:
                   type: http
                   url: "https://raw.githubusercontent.com/pingcap/tidb/{{ .Git.ref }}/br/metrics/alertmanager/lightning.rules.yml"
+              - name: prometheus/tiproxy.rules.yml
+                src:
+                  type: http
+                  # TODO: support building from tag.
+                  url: https://raw.githubusercontent.com/pingcap/monitoring/{{ .Git.ref }}/monitor-snapshot/{{ .Git.ref }}/operator/rules/tiproxy.rules.yml
```
Comment (wuhuizuo) [inline packages/packages.yaml.tmpl:268]: @xhebox PTAL

This file will be updated automatic when tiproxy repo changes.

File: packages/packages.yaml.tmpl [hunk 2]
```diff
@@ -261,22 +261,27 @@ components:
...
               - name: prometheus/kafka.rules.yml
                 src:
                   type: http
-                  url: "https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/kafka.rules.yml"
+                  url: "https://raw.githubusercontent.com/pingcap/monitoring/{{ .Git.ref }}/platform-monitoring/ansible/rule/kafka.rules.yml"
               - name: prometheus/node.rules.yml
                 src:
                   type: http
-                  url: "https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/node.rules.yml"
+                  url: "https://raw.githubusercontent.com/pingcap/monitoring/{{ .Git.ref }}/platform-monitoring/ansible/rule/node.rules.yml"
```
Comment (wuhuizuo) [inline packages/packages.yaml.tmpl:284]: @mornyx PTAL

fixed the paths.

**Comment by wuhuizuo** [comment]
> > **I have already done a preliminary review for you, and I hope to help you do a better job.**
> > The pull request fixes some URLs in the `packages.yaml.tmpl` file by updating the paths to the rule files for Prometheus monitoring. The changes are straightforward and do not introduce any new functionality.
> 
> There are no potential problems in this pull request. However, there is a TODO comment in the code that suggests adding support for building from tag.
> 
> To fix the TODO comment, the developer can add support for building from tag in the `packages.yaml.tmpl` file. They can do this by updating the `url` field in the `src` block to include the tag reference. For example, `url: https://raw.githubusercontent.com/pingcap/monitoring/{{ .Git.ref }}/monitor-snapshot/{{ .Git.ref }}/operator/rules/tiproxy.rules.yml` can be updated to `url: https://raw.githubusercontent.com/pingcap/monitoring/{{ .Git.ref }}/monitor-snapshot/v1.0.0/operator/rules/tiproxy.rules.yml`, where `v1.0.0` is the tag reference.
> 
> Overall, the changes in this pull request are simple and straightforward, and the developer can proceed with merging it.

But the tag is not constant, should be decided from `.Git.ref`.

**Review by xhebox — APPROVED** [review]

File: packages/packages.yaml.tmpl (hunk 2 above)
Comment (mornyx) [reply packages/packages.yaml.tmpl:284]: LGTM

**Pull request merged**